tests/python/unittest/test_operator.py

```diff
@@ -4541,6 +4541,22 @@ def get_large_matrix():
                 expected=[gt_topk(dat=a_npy, axis=1, ret_typ="mask", k=3,
                     is_ascend=True)])
 
+    def check_topk_small():
```

Comment:
Please add this to exiting for loop and use check_symbolic_forward and check_symbolic_backward